### PR TITLE
update(HTML): web/html/element/iframe

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: ["&lt;iframe&gt;: Елемент супутнього фрейму"@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/iframe), [сирці "&lt;iframe&gt;: Елемент супутнього фрейму"@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/iframe/index.md)

Нові зміни:
- [Remove self links, other pages (#35041)](https://github.com/mdn/content/commit/9231a7046973685f4600e1891fa644ecce41ef3b)
- [Convert noteblocks for web/html folder (#35114)](https://github.com/mdn/content/commit/9c09b183a5ce844a75c2f22e909d03f71ca329fc)
- [Convert noteblocks for web/html/element folder (#35085)](https://github.com/mdn/content/commit/b7955e77cd4293adf45ef23686df50b0305f02ad)
- [fix(html): update positioning and scaling section (#34529)](https://github.com/mdn/content/commit/71f57bc7f6065d835a5af7c3ced3ef26263c809f)
- [Add example of using srcdoc in iframe (#34171)](https://github.com/mdn/content/commit/38a8ede7b933ee8fdf35763b90edfa10cd7b561f)
- [Move Accessibility section above Examples for html element pages (#34007)](https://github.com/mdn/content/commit/bde0cb215d1d307c08678abe6623fc0d39f4cf7f)
- [Remove allow-downloads-without-user-activation sandbox value (#33409)](https://github.com/mdn/content/commit/a80ebc095216a642a96995c65833d3792b7b092d)

### CODE REVIEW

File: files/uk/web/html/element/iframe/index.md [hunk 1]
```diff
@@ -130,9 +128,17 @@ browser-compat: html.elements.iframe
...
 
 - `src`
+
   - : URL сторінки, що вбудовується. Щоб вбудувати порожню сторінку, котра відповідає [політиці того самого походження](/uk/docs/Web/Security/Same-origin_policy#uspadkovani-pokhodzhennia), слід використовувати значення `about:blank`. Також слід звернути увагу, що програмне усування атрибута `<iframe>` src (наприклад, за допомогою {{domxref("Element.removeAttribute()")}}) призводить до того, що в Firefox (починаючи від версії 65), браузерах на основі Chromium та Safari/iOS буде завантажено `about:blank`.
+
+  > **Примітка:** Сторінка `about:blank` використовує URL документа, щ вбудовується, як свій базовий URL, коли розв'язуються відносні URL, наприклад, якірні посилання.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/iframe/index.md:134]:
```suggestion
  > **Примітка:** Сторінка `about:blank` використовує URL документа, що вбудовується, як свій базовий URL, коли розв'язуються відносні URL, наприклад, якірні посилання.
```

File: files/uk/web/html/element/iframe/index.md [hunk 2]
```diff
@@ -130,9 +128,17 @@ browser-compat: html.elements.iframe
...
+
 - `srcdoc`
-  - : Супутній HTML для вбудовування, що відкидає атрибут `src`. Якщо браузер не підтримує атрибут `srcdoc`, то скористається URL в атрибуті `src`.
+
+  - : Супутній HTML для вбудовування, що відкидає атрибут `src`. Його вміст повинен відповідати синтаксису цілого документа URL, що складається з директиви doctype, тегів `<html>`, `<body>` тощо, хоч і більшість з цих тегів можна пропустити, залишивши лише вміст тіла. Такий документ матиме адресу `about:srcdoc`. Якщо браузер не підтримує атрибут `srcdoc`, то звернеться до URL у атрибуті `src`.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/iframe/index.md:138]:
```suggestion
  - : Супутній HTML для вбудовування, що відкидає атрибут `src`. Його вміст повинен відповідати синтаксису цілого документа URL, що складається з директиви doctype, тегів `<html>`, `<body>` тощо, хоч і більшість з цих тегів можна пропустити, залишивши лише вміст тіла. Такий документ матиме адресу `about:srcdoc`. Якщо браузер не підтримує атрибут `srcdoc`, то звернеться до URL в атрибуті `src`.
```

File: files/uk/web/html/element/iframe/index.md [hunk 3]
```diff
@@ -130,9 +128,17 @@ browser-compat: html.elements.iframe
...
+
+    > **Примітка:** Сторінка `about:srcdoc` використовує URL документа, щ вбудовується, як свій базовий URL, коли розв'язуються відносні URL, наприклад, якірні посилання.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/iframe/index.md:140]:
```suggestion
    > **Примітка:** Сторінка `about:srcdoc` використовує URL документа, що вбудовується, як свій базовий URL, коли розв'язуються відносні URL, наприклад, якірні посилання.
```

File: files/uk/web/html/element/iframe/index.md [hunk 4]
```diff
@@ -200,17 +218,45 @@ browser-compat: html.elements.iframe
...
+    height="250"
+></iframe>
+</article>
 ```
 
-Без цього `title` їм доведеться перейти до `<iframe>`, щоб з'ясувати, який вміст вбудовано. Така зміна контексту може спантеличувати й забирати час, особливо для сторінок з багатьма `<iframe>` чи коли вбудований інтерактивний вміст, як то відео чи аудіо.
+Ось як записувати при використанні `srcdoc` послідовності екранування:
+
+- По-перше, випишіть HTML, екрануючи все, що екранувалося б у звичайному документі HTML (наприклад, `<`, `>`, `&` тощо).
+- `&lt;` і `<` у атрибуті `srcdoc` представляють один і той же символ. Таким чином, щоб зробити таку послідовність справжньою послідовністю екранування в документі HTML, замініть усі амперсанди (`&`) на `&amp;`. Наприклад, `&lt;` стане `&amp;lt;`, а `&amp;` стане `&amp;amp;`.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/iframe/index.md:254]:
```suggestion
- `&lt;` і `<` в атрибуті `srcdoc` представляють один і той же символ. Таким чином, щоб зробити таку послідовність справжньою послідовністю екранування в документі HTML, замініть усі амперсанди (`&`) на `&amp;`. Наприклад, `&lt;` стане `&amp;lt;`, а `&amp;` стане `&amp;amp;`.
```

File: files/uk/web/html/element/iframe/index.md [hunk 5]
```diff
@@ -200,17 +228,45 @@ browser-compat: html.elements.iframe
 
 {{EmbedLiveSample('prostyi-iframe', 640,400)}}
 
-## Занепокоєння щодо доступності
+### Вбудовування в \<iframe> вихідного коду
 
-Люди, що орієнтуються за допомогою допоміжної технології, як то читач з екрана, можуть використовувати [атрибут `title`](/uk/docs/Web/HTML/Global_attributes/title) на `<iframe>` як позначення його вмісту. Значення `title` повинно стисло описувати вбудований вміст:
+Цей приклад безпосередньо візуалізує вихідний код у супутньому фреймі. Це можна використовувати як техніку для запобігання ін'єкціям сценаріїв при виведенні вмісту, створеного користувачами, коли поєднати це з атрибутом `sandbox`.
```
Comment (alinkedd) [inline files/uk/web/html/element/iframe/index.md:233]:
```suggestion
Цей приклад безпосередньо візуалізує вихідний код у супутньому фреймі. Якщо поєднати з атрибутом `sandbox`, то це можна використовувати як техніку для запобігання ін'єкціям сценаріїв при виведенні вмісту, створеного користувачами.
```

File: files/uk/web/html/element/iframe/index.md [hunk 6]
```diff
@@ -200,17 +228,45 @@ browser-compat: html.elements.iframe
...
 
-```html
-<iframe
-  title="Сторінка Вікіпедії про авокадо"
-  src="https://uk.wikipedia.org/wiki/%D0%90%D0%B2%D0%BE%D0%BA%D0%B0%D0%B4%D0%BE"></iframe>
+Зверніть увагу на те, що при використанні `srcdoc` будь-які відносні URL у вбудованому вмісті розв'язуються відносно URL вбудованої сторінки. При потребі використання якірних посилань, що вказують на місця у вбудованому вмісті, необхідно явно задати `about:srcdoc` як базовий URL.
```
Comment (alinkedd) [inline files/uk/web/html/element/iframe/index.md:235]:
```suggestion
Зверніть увагу на те, що за використання `srcdoc` будь-які відносні URL у вбудованому вмісті розв'язуються відносно URL вбудованої сторінки. Якщо потрібно використати якірні посилання, що вказують на місця у вбудованому вмісті, необхідно явно задати `about:srcdoc` як базовий URL.
```

File: files/uk/web/html/element/iframe/index.md [hunk 7]
```diff
@@ -200,17 +228,45 @@ browser-compat: html.elements.iframe
...
+    height="250"
+></iframe>
+</article>
 ```
 
-Без цього `title` їм доведеться перейти до `<iframe>`, щоб з'ясувати, який вміст вбудовано. Така зміна контексту може спантеличувати й забирати час, особливо для сторінок з багатьма `<iframe>` чи коли вбудований інтерактивний вміст, як то відео чи аудіо.
+Ось як записувати при використанні `srcdoc` послідовності екранування:
+
+- По-перше, випишіть HTML, екрануючи все, що екранувалося б у звичайному документі HTML (наприклад, `<`, `>`, `&` тощо).
+- `&lt;` і `<` в атрибуті `srcdoc` представляють один і той же символ. Таким чином, щоб зробити таку послідовність справжньою послідовністю екранування в документі HTML, замініть усі амперсанди (`&`) на `&amp;`. Наприклад, `&lt;` стане `&amp;lt;`, а `&amp;` стане `&amp;amp;`.
```
Comment (alinkedd) [inline files/uk/web/html/element/iframe/index.md:264]:
```suggestion
- `&lt;` і `<` в атрибуті `srcdoc` представляють один і той же символ. Тому, щоб зробити таку послідовність справжньою послідовністю екранування в документі HTML, замініть усі амперсанди (`&`) на `&amp;`. Наприклад, `&lt;` стане `&amp;lt;`, а `&amp;` – `&amp;amp;`.
```

File: files/uk/web/html/element/iframe/index.md [hunk 8]
```diff
@@ -25,23 +26,26 @@ browser-compat: html.elements.iframe
...
-    > **Примітка:** Цей атрибут вважається застарілим, він був перевизначений як `allow="fullscreen"`.
+    > [!NOTE]
+    > Цей атрибут вважається застарілим, він був перевизначений як `allow="fullscreen"`.
 
 - `allowpaymentrequest` {{deprecated_inline}} {{non-standard_inline}}
 
   - : Слід надати цьому атрибутові значення `true`, якщо `<iframe>` до іншого походження треба дозволити викликати [API запитів платежів](/uk/docs/Web/API/Payment_Request_API).
 
-    > **Примітка:** Цей атрибут вважається застарілим, він був перевизначений як `allow="payment"`.
+    > [!NOTE]
```
Comment (alinkedd) [inline files/uk/web/html/element/iframe/index.md:43]: треба ще цей note додати https://github.com/mdn/content/blob/main/files/en-us/web/html/element/iframe/index.md?plain=1#L72-L74 для loading lazy

**Review by alinkedd — CHANGES_REQUESTED** [review]

File: files/uk/web/html/element/iframe/index.md (hunk 8 above)
Comment (undead404) [reply files/uk/web/html/element/iframe/index.md:43]: Додано

**Review by alinkedd — APPROVED** [review]

**Pull request merged**